metricbeat/module/windows/service/_meta/fields.yml

```diff
@@ -10,13 +10,13 @@
       description: >
         A unique ID for the service. It is a hash of the machine's GUID and the
         service name.
-
+    # ECS?
     - name: name
```

Comment:
Yes, service name, and below display_name => service title (not in ECS yet, but sounds like same semantics as process.title)?

Reply:
Skipping for now, will not get to it.